x/gamm/keeper/migrate_test.go

```diff
@@ -716,3 +716,57 @@ func (suite *KeeperTestSuite) TestGetLinkedConcentratedPoolID() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestGetLinkedBalancerPoolID() {
+	tests := []struct {
+		name                  string
+		poolIdEntering        []uint64
+		expectedPoolIdLeaving []uint64
+		expectErr             bool
+	}{
+		{
+			name:                  "Happy path",
+			poolIdEntering:        []uint64{4, 5, 6},
+			expectedPoolIdLeaving: []uint64{1, 2, 3},
+			expectErr:             false,
+		},
+		{
+			name:           "error: set poolIdEntering to a balancer pool ID",
+			poolIdEntering: []uint64{3},
+			expectErr:      true,
+		},
+		{
+			name:           "error: set poolIdEntering to a non existent pool ID",
+			poolIdEntering: []uint64{7},
+			expectErr:      true,
+		},
```

Comment:
We should add a test case where we skip setting migration info and attempt to retrieve the happy path (pools exist, just not the record)

Reply:
Bump